src/test/cli/ceph-conf/show-config-value.t

```diff
@@ -1,6 +1,6 @@
 
-# should reflect daemon defaults
-
+Should reflect daemon defaults
```

Comment:
i don't think this is a document rendered by sphinx. could you keep the `#` which marks the beginning of a comment?

Reply:
Sure.